src/clusterfuzz/_internal/cron/cleanup.py

```diff
@@ -1070,6 +1072,11 @@ def update_component_labels(policy, testcase, issue):
   for filtered_component in filtered_components:
     issue.components.add(filtered_component)
 
+  # This is very specific to google_issue_tracker, so this attribute might not
+  # be available for other issue trackers.
+  if component_id and hasattr(issue, 'component_id'):
+    issue.component_id = component_id
```

Comment:
Would this attribute assignment explode for other issue trackers, or would the assigned value simply be ignored?  If the latter, we can set it regardless of whether issue has a `component_id` attribute yet.

Reply:
No, that would explode, which is why it's conditioned.